components/html/modules/header/_siteName.html

```diff
@@ -2,7 +2,11 @@
 	<div class="container">
 		<div class="row">
 			<div class="logo col-lg-2 col-md-3 col-sm-3 col-xs-12">
-				<a href="https://www.purdue.edu"><img alt="Purdue University" src="https://www.purdue.edu/purdue/images/logo.png"/></a>
+				<a class="svgLinkContainer" href="https://www.purdue.edu">
```

Comment:
When do you both, @bholaday @jengarza, think it is an appropriate time to start switching to using ITCSS + BEM? I’m looking at our class names for some of the new elements and it’s following a similar naming convention to what we've been doing. It's literal, describing the element's purpose but in no way uniform. We could take this opportunity to slowly update various portions of the templates, but then we could only reliably send this update to clients using the newer base project within Cascade.

Reply:
Yeah, we'll need to think about how we want to implement it - will we need a legacy stylesheet people can reference?

Reply:
Unfortunately, I think we always will. Using Cascade means having to publish, and sometimes deploy, any changes we make at a global level. Since we can't have everybody do that at the same time when a global change is made, we have to keep old versions of stylesheets to support any page that isn't republished.